app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <application
+        android:label="@string/app_name_debug"
```

Comment:
Why would you add `app_name_debug`? We can override `app_name` and avoid using one more `AndroidManifest.xml` file

Reply:
I have tried using `app_name`, but it will change along with the text in the toolbar. Since the title of LogFox is centered, this can cause the title of the home page to be compressed due to its length, making it less attractive.

Reply:
OK then, leave it this way. But i'd like the debug app name to be `LogFox Debug`